src/styles/blocks/page.scss

```diff
@@ -0,0 +1,10 @@
+.page {
+
+  &__header {
+    margin-bottom: 50px;
+  }
+
+  &__catalog {
+    margin: auto;
+  }
```

Comment:
try to add margin-top to catalogue instead of margin-bottom to header for the tests

```suggestion
  &__catalog {
    margin: 50px auto;
  }
```

Reply:
It doesn't work either (